GraphicEditor/View/Windows/RoomInformation.xaml.cs

```diff
@@ -23,6 +23,7 @@ public partial class RoomInformation : Window
         public RoomInformation()
         {
             this.DataContext = RoomInformationViewModel.GetInstance();
+            RoomInformationViewModel.GetInstance().window = this;
```

Comment:
Any way we could avoid singleton pattern? 😄  Especially since this is going to need tests

Reply:
I agree with this. I only made this thing to work. I wanted to clean this mess up, but gave you here just a preview what's going on. This will be solved today.